lib/IR/GraphScheduler.h

```diff
@@ -21,6 +21,12 @@
 #include <unordered_map>
 
 namespace glow {
+
```

Comment:
Doxygen:
/// Specifies the kind of graph scheduling to perform. 

/// This is a heuristics that tries to minimize memory usage.

/// Performs a standard topological search

Reply:
Done